portal/static/js/src/data/common/ExportInstrumentsData.js

```diff
@@ -21,5 +21,7 @@ export default {
         dataType: "csv",
         selected: "",
         message: ""
-    }
+    },
+    requestSubmittedDisplay: i18next.t("Export request submitted"),
+    failedRequestDisplay: i18next.t("Request to export report data failed.")
```

Comment:
```suggestion
    failedRequestDisplay: i18next.t("Request to export data failed.")
```
Would prefer to use more consistent and generic language for errors/warnings- they're easier to reuse and less of a i18n management burden.